test/blackbox-tests/test-cases/configurator-import.t/dune

```diff
@@ -0,0 +1,12 @@
+(rule
+ (targets c_flags.sexp)
+ (deps
+  (:deps-conf configure/configure.exe))
+ (action
+  (run %{deps-conf})))
+
+(executable
+ (name configurator)
+ (flags
+  :standard
+  (:include c_flags.sexp)))
```

Comment:
Do we need this binary? Isn't it enough to just build `c_flags.sexp` in the test?

Reply:
I did not know we could just build the sexp